prac_06/guitars.py

```diff
@@ -0,0 +1,42 @@
+"""
+Practical 06 - Samuel Barrett - 13038579
+User Guitar listing with class
+"""
+
+from prac_06.guitar import Guitar
+
+
+def main():
+    """Program to manage a guitar collection"""
+
+    guitar_names = []
+    guitar_total = int(input("How many guitars do you want to list: "))
```

Comment:
This wasn't the program design. `guitar_total` is not a good name for a `number_of_guitars`.